tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -579,6 +579,12 @@ iface_loopback_action/test_iface_loopback_action.py:
 #######################################
 #####      iface_namingmode       #####
 #######################################
+iface_namingmode/test_iface_namingmode.py::TestConfigInterface::test_config_interface_speed:
```

Comment:
Have you validated that? seems it will still impact subsequent cases if just skip test_config_interface_speed.

Reply:
I ran iface_namingmode/test_iface_namingmode.py::TestConfigInterface and did not see any issue. 
---------------------------------- generated xml file: /run_logs/154-TestConfigInterface/iface_namingmode/test_iface_namingmode.py::TestConfigInterface_2023-11-14-11-15-59.xml -----------------------------------
INFO:root:Can not get Allure report URL. Please check logs
--------------------------------------------------------------------------------------------- live log sessionfinish ----------------------------------------------------------------------------------------------
11:18:52 __init__.pytest_terminal_summary         L0064 INFO   | Can not get Allure report URL. Please check logs
============================================================================================= short test summary info =============================================================================================
SKIPPED [2] iface_namingmode/test_iface_namingmode.py: x86_64-8111_32eh_o-r0 platform does not support TestConfigInterface test config interface speed
=============================================================================== 4 passed, 2 skipped, 1 warning in 171.94s (0:02:51) 
I also ran the iface_namingmode/test_iface_namingmode.py with and without skip and below are the result
without skip
----------

-------------------------------------------------------------------------- generated xml file: /run_logs/aaa/tr_2023-11-18-06-34-28.xml ---------------------------------------------------------------------------
INFO:root:Can not get Allure report URL. Please check logs
--------------------------------------------------------------------------------------------- live log sessionfinish ----------------------------------------------------------------------------------------------
06:50:36 __init__.pytest_terminal_summary         L0064 INFO   | Can not get Allure report URL. Please check logs
============================================================================================= short test summary info =============================================================================================
SKIPPED [4] iface_namingmode/test_iface_namingmode.py:640: Unsupported topology
========================================================================= 56 passed, 4 skipped, 1 warning in 966.78s (0:16:06) ====================


with skip
--------

------------------------------------------------------------------------- generated xml file: /run_logs/aaa-s/tr_2023-11-18-06-53-53.xml --------------------------------------------------------------------------
INFO:root:Can not get Allure report URL. Please check logs
--------------------------------------------------------------------------------------------- live log sessionfinish ----------------------------------------------------------------------------------------------
07:05:01 __init__.pytest_terminal_summary         L0064 INFO   | Can not get Allure report URL. Please check logs
============================================================================================= short test summary info =============================================================================================
SKIPPED [4] iface_namingmode/test_iface_namingmode.py:640: Unsupported topology
SKIPPED [2] iface_namingmode/test_iface_namingmode.py: x86_64-8111_32eh_o-r0 platform does not support TestConfigInterfacei test config interface speed
========================================================================= 54 passed, 6 skipped, 1 warning in 666.84s (0:11:06) ==========================================================================